nextcord/application_command.py

```diff
@@ -92,6 +93,13 @@
 else:
     _CustomTypingMetaBase = object
 
+if sys.version_info[1] >= 10:
```

Comment:
This is more specific and readable. Also supported by conditional Pyright checks I believe (though I'm not 100% sure).

```suggestion
if sys.version_info >= (3, 10):
```

Reply:
Tested on 3.10 and 3.8 and commited.